src/e2e/java/teammates/e2e/util/ResultsStatistics.java

```diff
@@ -0,0 +1,75 @@
+package teammates.e2e.util;
+
+/**
+ * Result statistics class for L&P tests.
+ */
+public class ResultsStatistics {
+
+    private double meanResTime;
+    private double pct1ResTime; // 90th percentile
+    private double throughput;
+    private int errorCount;
```

Comment:
Let's use the `errorPct` field instead of this and remove the getErrorRate() method?

Reply:
Anyway due to the `gson.fromJson(endPointAnalysis, LNPResultsStatistics.class)` method, the fields need to map exactly as the `json` file generated! So I can only post-process the data instead :(

Reply:
@jacoblipech if you want to achieve what @amrut-prabhu suggested, you're looking for `@SerializedName` annotation.

Reply:
Doesn't the statistics.json file have a field called `errorPct`?

Reply:
ohh saw it! But I thought we discussed previously to show the number of error tests fails and the percentage?
`+ ",  Err: " + errorCount + " (" + String.format("%.2f", getErrorPercentage()) + "%)";`

Reply:
Oh, yeah... Then just add `errorPct` (and remove the method). No need to remove `errorCount`. Sorry for the confusion 😅